### ISSUE TITLE
Padding around FlexContainer is too small

### ISSUE THREAD

**Issue by jerome-obeo** [eclipse-sirius/sirius-web#2161]
* [x] **I have checked that this bug has not yet been reported by someone else**
* [x] **I have checked that this bug appears on Chrome**
* [x] **I have specified the version** :
 6.4

* [x] **I have specified my environment** : 
- All

### Screenshots
![image](https://github.com/eclipse-sirius/sirius-components/assets/128463513/d7940632-dfd1-4281-b9f6-1404caa08f1f)

### Steps to reproduce

1. Create a `org.eclipse.sirius.components.view.form.FlexboxContainerDescription` with a border style
2. Add some content inside this container

### Expected behavior

Space between the content of the container and the border should be increased.
Could be 8px for instance:
![image](https://github.com/eclipse-sirius/sirius-components/assets/128463513/e01cfb45-a8bb-4680-bbac-ffcdc4f56ee5)

### Actual behavior

At the moment, FlexContainer has a padding of 4px:
![image](https://github.com/eclipse-sirius/sirius-components/assets/128463513/1a1191e0-c9c7-417b-8231-de5bff52a98c)

**Comment by jerome-obeo** [comment]
Can be seen on live demo of papyrus web at [https://www.demo-papyrusweb.fr](https://www.demo-papyrusweb.fr/)

1. Open `All concepts` project
2. Unfold Explorer tree until `MemberEndAssociation`  is revealed (it is underneath `all-concepts` node).
3. Select  `MemberEndAssociation` node
4. Choose UML page in Details panel

Member End groups should be displayed

**Comment by sbegaudeau** [comment]
I don’t think that changing the default value this way would be wise. It should not have any padding at all by default and instead it should expose at least the following properties to the specifier:

- padding
- gap
- alignItems
- justifyContent

because we will end up with countless of those issues were people are expecting us to tweak the default values for their specific use case. We should give you control other that and have CSS default values as the default. This way, it’s between you and CSS :)

**Comment by jerome-obeo** [comment]
Sounds great. You are right, more control on style is the best approach.
